subsys/bluetooth/host/Kconfig

```diff
@@ -436,12 +440,13 @@ config BT_PERIPHERAL_PREF_SLAVE_LATENCY
 config BT_PERIPHERAL_PREF_TIMEOUT
 	int "Peripheral preferred supervision timeout in 10ms units"
 	default 42
-	range 10 3200
+	range 0 3200
```

Comment:
I think it'd have been better to change the maximum to 0xffff (the decimal equivalent) instead. That way you don't introduce any code size or runtime overhead (now your solution is adding an an extra branch to each assignment).

Reply:
Is there a way to have multiple ranges in Kconfig? @ulfalizer. Eg:

```
range 0 3200
range 0xffff 0xffff
```

Reply:
That'd indeed be nice.

Reply:
I though that would not be possible, only the last one will be used.

Reply:
@Vudentz that's what I think as well, however it's a feature worth considering since we're in control of our Kconfig parser. Another possible syntax is to have multiple ranges in a single statement, e.g. `range 10 3200 0xffff 0xfff`. Btw that also gives the idea that we could have a flexible type that accepts both decimal and hex input rather than the current int/hex which only accept one or the other.